src/components/LandingFooter/index.js

```diff
@@ -4,7 +4,11 @@ import './LandingFooter.css';
 const LandingFooter = () => (
   <footer className="LandingFooter">
     <p className="LandingFooterCopyright">
-      Copyright © 2020 Crane Cloud. All Rights Reserved.
+      Copyright &copy;
```

Comment:
better, but also i hate the copyright mark - most typefaces don't implement it well - it looks off, i suggest we remove it

Reply:
@kawerewagaba cool let me do that.